core/src/test/java/org/elasticsearch/common/network/NetworkModuleTests.java

```diff
@@ -163,32 +160,6 @@ public void testRegisterHttpTransport() {
         assertFalse(module.isTransportClient());
     }
 
-    public void testRegisterRestHandler() {
```

Comment:
I didn't sweat losing them because we exercise it so much with plugins. We actually already have a test that relies on much of this behavior but I'll make similar tests.